lib/route_downcaser.rb

```diff
@@ -6,5 +6,6 @@ module RouteDowncaser
   extend RouteDowncaser::Configuration
 
   define_setting :redirect, false
+  define_setting :include_patterns
```

Comment:
Should we initialize this as an empty array and use ```RouteDowncaser.include_patterns.any?``` instead of ```RouteDowncaser.include_patterns``` on lib/route_downcaser/downcase_route_middleware.rb#57?